src/CodeGen_Internal.cpp

```diff
@@ -459,7 +459,11 @@ std::unique_ptr<llvm::TargetMachine> make_target_machine(const llvm::Module &mod
                                                 mcpu, mattrs,
                                                 options,
                                                 llvm::Reloc::PIC_,
+#ifdef HALIDE_USE_CODEMODEL_LARGE
+                                                llvm::CodeModel::Large,
```

Comment:
Worth adding a comment here to the effect of "Halide is not regularly tested with CodeModel::Large; this is is provided for experimentation and may break"